packages/edit-post/src/components/layout/index.native.js

```diff
@@ -15,6 +15,7 @@ import {
 	HTMLTextInput,
 	KeyboardAvoidingView,
 	NoticeList,
+	AutocompletionItemsSlot,
```

Comment:
It looks like these are new APIs. Could they be marked unstable?

Reply:
Good idea. Thanks for the suggestion!

Reply:
Updated https://github.com/WordPress/gutenberg/pull/29772/commits/756c056215227fa1b2b3ae99caed4e6e25807142